data_validation/query_builder/query_builder.py

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
Can we delete the query_builder/custom_query_builder.py file with this refactor?

Reply:
Sure. We'll also need to remove the testcase associated with it (tests/unit/query_builder/test_custom_query_builder.py), otherwise the nox test will fail.

We'll anyway add a new test case for custom query later, so I think it should be okay. I'll remove both the custom_query_builder.py as well as the test.
Does that sound good?

Reply:
Deleted the py file and its testcase in this [commit](https://github.com/GoogleCloudPlatform/professional-services-data-validator/commit/f5b17d9760f8748c42852ed7ce30767344bbd020)